libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -48,9 +48,9 @@ using namespace solidity;
 using namespace solidity::util;
 using namespace solidity::frontend;
 
-pair<string, string> IRGenerator::run(ContractDefinition const& _contract)
+pair<string, string> IRGenerator::run(ContractDefinition const& _contract, string _dependanciesSource)
```

Comment:
```suggestion
pair<string, string> IRGenerator::run(ContractDefinition const& _contract, string _dependenciesSource)
```
Same typo lower in this file